### DISCUSSION TITLE
Llama Vision Model Locally doesn't work, works fine with Groq

### DISCUSSION THREAD

**Discussion by RamboRogers** [open-webui/open-webui#6829] [General]
# Bug Report

<img width="817" alt="image" src="https://github.com/user-attachments/assets/459de947-06a0-4970-a927-8cda661d7109">

## Important Notes

- **Before submitting a bug report**: Please check the Issues or Discussions section to see if a similar issue or feature request has already been posted. It's likely we're already tracking it! If you’re unsure, start a discussion post first. This will help us efficiently focus on improving the project.

- **Collaborate respectfully**: We value a constructive attitude, so please be mindful of your communication. If negativity is part of your approach, our capacity to engage may be limited. We’re here to help if you’re open to learning and communicating positively. Remember, Open WebUI is a volunteer-driven project managed by a single maintainer and supported by contributors who also have full-time jobs. We appreciate your time and ask that you respect ours.

- **Contributing**: If you encounter an issue, we highly encourage you to submit a pull request or fork the project. We actively work to prevent contributor burnout to maintain the quality and continuity of Open WebUI.

- **Bug reproducibility**: If a bug cannot be reproduced with a `:main` or `:dev` Docker setup, or a pip install with Python 3.11, it may require additional help from the community. In such cases, we will move it to the "issues" Discussions section due to our limited resources. We encourage the community to assist with these issues. Remember, it’s not that the issue doesn’t exist; we need your help!

Note: Please remove the notes above when submitting your post. Thank you for your understanding and support!

---

## Installation Method

[Describe the method you used to install the project, e.g., git clone, Docker, pip, etc.]

## Environment

- **Open WebUI Version:** [e.g., v0.3.11]
v0.3.35
[(latest)](https://github.com/open-webui/open-webui/releases/tag/v0.3.35)
- **Ollama (if applicable):** [e.g., v0.2.0, v0.1.32-rc1]
[matt@leviathan ~]$ ollama --version
ollama version is 0.4.1

- **Operating System:** [e.g., Windows 10, macOS Big Sur, Ubuntu 20.04]
- **Browser (if applicable):** [e.g., Chrome 100.0, Firefox 98.0]
Chrome

**Confirmation:**

- [x ] I have read and followed all the instructions provided in the README.md.
- [x ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [ ] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

[Describe what you expected to happen.]

## Actual Behavior:

[Describe what actually happened.]

## Description

**Bug Summary:**
[Provide a brief but clear summary of the bug]

## Reproduction Details

**Steps to Reproduce:**
[Outline the steps to reproduce the bug. Be as detailed as possible.]

Upload an image to openwebui and ask a question about it.

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

INFO:     192.168.1.52:11354 - "GET /static/favicon.png HTTP/1.1" 304 Not Modified
INFO:     192.168.1.52:48348 - "GET /static/favicon.png HTTP/1.1" 304 Not Modified
INFO:     192.168.1.52:40155 - "POST /api/v1/memories/query HTTP/1.1" 200 OK
INFO  [open_webui.apps.ollama.main] url: http://192.168.2.97:11434
INFO:     192.168.1.52:40155 - "POST /ollama/api/chat HTTP/1.1" 500 Internal Server Error
INFO:     192.168.1.52:40155 - "POST /api/v1/chats/17653728-730b-4f5b-a1a3-127e7833a9c0 HTTP/1.1" 200 OK
INFO:     192.168.1.52:40155 - "GET /api/v1/chats/?page=1 HTTP/1.1" 200 OK

**Screenshots/Screen Recordings (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by acn-bart** [comment]
I have the same with those versions and on macOS 15.1. This looks more of issue than discussion, but for some reason somebody converted that from issue ;-)

**Comment by yitang** [comment]
i'm having the same issues in Debian. ollama version is 0.4.1

**Reply by JS-an** [reply]
i'm having the same issues in ubuntu. ollama version is 0.4.4
